include/CoolPropLib.h

```diff
@@ -270,6 +270,16 @@
      * @return A handle to the state class generated
      */
     EXPORT_CODE long CONVENTION AbstractState_factory(const char* backend, const char* fluids, long *errcode, char *message_buffer, const long buffer_length);
+    /**
+     * @brief Get the fluid names for the AbstractState
+     * @param handle The integer handle for the state class stored in memory
+     * @param fluids '&' delimited list of fluids
```

Comment:
It would be better if you used the delimiter from the configuration: http://www.coolprop.org/coolprop/Configuration.html (LIST_STRING_DELIMITER)